releasenotes/notes/inverse_cancellation_no_python_path-6b0f254a649d2af2.yaml

```
@@ -0,0 +1,25 @@
+---
+features_transpiler:
+  - |
+    Added a default set of inverse gates for the :class:`.InverseCancellation`
+    transpiler pass. Previously, an explicit list of gates or gate pairs to
+    cancel was a required argument for the constructor of the pass object.
+    Now this list is optional and if no list is provided the self inverse gates:
```

```suggestion
    Now this list is optional and if no list is provided the self inverse gates are:
```

Should we also stress that not providing the optional list is now the default?

---

It probably wouldn't hurt. I can reword it a bit, or we can save it for the release PR too.